spring-cloud-sleuth-zipkin-stream/src/main/java/org/springframework/cloud/sleuth/zipkin/stream/SamplingZipkinSpanIterator.java

```diff
@@ -38,6 +41,11 @@
  * @since 1.0.0
  */
 final class SamplingZipkinSpanIterator implements Iterator<zipkin.Span> {
+	private static final List<String> ZIPKIN_ANNOTATIONS = Arrays.asList(
```

Comment:
I don't think the _ADDR things need to be here, as I don't think a sleuth span would add these as tags, right? I think Span.SPAN_PEER_SERVICE_TAG_NAME is what's being tagged instead now